docker-compose-dev.yml

```diff
@@ -164,24 +163,24 @@ services:
     tmpfs:
       - /mail:mode=770,size=128M,uid=1000,gid=1000
 
-  dev_keycloak:
-    container_name: "dev_keycloak"
-    image: 'quay.io/keycloak/keycloak:21.0'
+  keycloak:
+    container_name: keycloak
+    image: keycloak/keycloak:25.0
     hostname: keycloak
+    command:
+      - 'start-dev'
+      - '--http-port=9080'
+      - '--import-realm'
+    volumes:
+      - ./keycloak:/opt/keycloak/data/import
     environment:
-      - KEYCLOAK_ADMIN=kcadmin
-      - KEYCLOAK_ADMIN_PASSWORD=kcpassword
-      - KEYCLOAK_LOGLEVEL=DEBUG
-      - KC_HOSTNAME_STRICT=false
-    networks:
-      dataverse:
-        aliases:
-          - keycloak.mydomain.com #create a DNS alias within the network (add the same alias to your /etc/hosts to get a working OIDC flow)
-    command: start-dev --import-realm --http-port=8090  # change port to 8090, so within the network and external the same port is used
+      KC_HTTP_PORT: 9080
+      KEYCLOAK_ADMIN: admin@kuleuven.be
+      KEYCLOAK_ADMIN_PASSWORD: password
     ports:
-      - "8090:8090"
-    volumes:
-      - './conf/keycloak/test-realm.json:/opt/keycloak/data/import/test-realm.json'
+      - 9080:9080
+    networks:
+     - dataverse
 
```

Comment:
I'm on cbde18f478 and getting a crazy error when I run this:

```
mvn -Pct clean package docker:run
```

Any ideas? I feel like I've seen errors like this when there's something wrong with the compose file but it looks fine to me. 🤔 

```

[INFO] DOCKER> [gdcc/configbaker:unstable] "dev_bootstrap": Stop and removed container 5944c579a147 after 0 ms
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  26.733 s
[INFO] Finished at: 2024-10-03T14:08:12-04:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal io.fabric8:docker-maven-plugin:0.45.0:run (default-cli) on project dataverse: A type incompatibility occurred while executing io.fabric8:docker-maven-plugin:0.45.0:run: class java.lang.Integer cannot be cast to class java.lang.String (java.lang.Integer and java.lang.String are in module java.base of loader 'bootstrap')
[ERROR] -----------------------------------------------------
[ERROR] realm =    plugin>io.fabric8:docker-maven-plugin:0.45.0
[ERROR] strategy = org.codehaus.plexus.classworlds.strategy.SelfFirstStrategy
[ERROR] urls[0] = file:/Users/PDurbin/.m2/repository/io/fabric8/docker-maven-plugin/0.45.0/docker-maven-plugin-0.45.0.jar
[ERROR] urls[1] = file:/Users/PDurbin/.m2/repository/com/github/jnr/jffi/1.3.11/jffi-1.3.11.jar
[ERROR] urls[2] = file:/Users/PDurbin/.m2/repository/com/github/jnr/jffi/1.3.11/jffi-1.3.11-native.jar
[ERROR] urls[3] = file:/Users/PDurbin/.m2/repository/com/github/jnr/jnr-unixsocket/0.38.19/jnr-unixsocket-0.38.19.jar
[ERROR] urls[4] = file:/Users/PDurbin/.m2/repository/com/github/jnr/jnr-ffi/2.2.13/jnr-ffi-2.2.13.jar
[ERROR] urls[5] = file:/Users/PDurbin/.m2/repository/org/ow2/asm/asm-commons/9.2/asm-commons-9.2.jar
[ERROR] urls[6] = file:/Users/PDurbin/.m2/repository/org/ow2/asm/asm-analysis/9.2/asm-analysis-9.2.jar
[ERROR] urls[7] = file:/Users/PDurbin/.m2/repository/org/ow2/asm/asm-tree/9.2/asm-tree-9.2.jar
[ERROR] urls[8] = file:/Users/PDurbin/.m2/repository/org/ow2/asm/asm-util/9.2/asm-util-9.2.jar
[ERROR] urls[9] = file:/Users/PDurbin/.m2/repository/com/github/jnr/jnr-a64asm/1.0.0/jnr-a64asm-1.0.0.jar
[ERROR] urls[10] = file:/Users/PDurbin/.m2/repository/com/github/jnr/jnr-x86asm/1.0.2/jnr-x86asm-1.0.2.jar
[ERROR] urls[11] = file:/Users/PDurbin/.m2/repository/com/github/jnr/jnr-constants/0.10.4/jnr-constants-0.10.4.jar
[ERROR] urls[12] = file:/Users/PDurbin/.m2/repository/com/github/jnr/jnr-enxio/0.32.14/jnr-enxio-0.32.14.jar
[ERROR] urls[13] = file:/Users/PDurbin/.m2/repository/com/github/jnr/jnr-posix/3.1.16/jnr-posix-3.1.16.jar
[ERROR] urls[14] = file:/Users/PDurbin/.m2/repository/com/google/cloud/tools/jib-core/0.24.0/jib-core-0.24.0.jar
[ERROR] urls[15] = file:/Users/PDurbin/.m2/repository/com/google/cloud/tools/jib-build-plan/0.4.0/jib-build-plan-0.4.0.jar
[ERROR] urls[16] = file:/Users/PDurbin/.m2/repository/com/google/http-client/google-http-client/1.42.2/google-http-client-1.42.2.jar
[ERROR] urls[17] = file:/Users/PDurbin/.m2/repository/io/opencensus/opencensus-api/0.31.1/opencensus-api-0.31.1.jar
[ERROR] urls[18] = file:/Users/PDurbin/.m2/repository/io/grpc/grpc-context/1.27.2/grpc-context-1.27.2.jar
[ERROR] urls[19] = file:/Users/PDurbin/.m2/repository/io/opencensus/opencensus-contrib-http-util/0.31.1/opencensus-contrib-http-util-0.31.1.jar
[ERROR] urls[20] = file:/Users/PDurbin/.m2/repository/com/google/http-client/google-http-client-apache-v2/1.42.2/google-http-client-apache-v2-1.42.2.jar
[ERROR] urls[21] = file:/Users/PDurbin/.m2/repository/com/google/auth/google-auth-library-oauth2-http/1.10.0/google-auth-library-oauth2-http-1.10.0.jar
[ERROR] urls[22] = file:/Users/PDurbin/.m2/repository/com/google/auto/value/auto-value-annotations/1.9/auto-value-annotations-1.9.jar
[ERROR] urls[23] = file:/Users/PDurbin/.m2/repository/com/google/auth/google-auth-library-credentials/1.10.0/google-auth-library-credentials-1.10.0.jar
[ERROR] urls[24] = file:/Users/PDurbin/.m2/repository/com/google/http-client/google-http-client-gson/1.42.2/google-http-client-gson-1.42.2.jar
[ERROR] urls[25] = file:/Users/PDurbin/.m2/repository/org/apache/commons/commons-compress/1.21/commons-compress-1.21.jar
[ERROR] urls[26] = file:/Users/PDurbin/.m2/repository/com/fasterxml/jackson/core/jackson-databind/2.15.0/jackson-databind-2.15.0.jar
[ERROR] urls[27] = file:/Users/PDurbin/.m2/repository/com/fasterxml/jackson/core/jackson-annotations/2.15.0/jackson-annotations-2.15.0.jar
[ERROR] urls[28] = file:/Users/PDurbin/.m2/repository/com/fasterxml/jackson/core/jackson-core/2.15.0/jackson-core-2.15.0.jar
[ERROR] urls[29] = file:/Users/PDurbin/.m2/repository/com/fasterxml/jackson/datatype/jackson-datatype-jsr310/2.15.0/jackson-datatype-jsr310-2.15.0.jar
[ERROR] urls[30] = file:/Users/PDurbin/.m2/repository/org/ow2/asm/asm/9.5/asm-9.5.jar
[ERROR] urls[31] = file:/Users/PDurbin/.m2/repository/com/google/code/gson/gson/2.8.9/gson-2.8.9.jar
[ERROR] urls[32] = file:/Users/PDurbin/.m2/repository/com/google/guava/guava/32.0.1-jre/guava-32.0.1-jre.jar
[ERROR] urls[33] = file:/Users/PDurbin/.m2/repository/com/google/guava/failureaccess/1.0.1/failureaccess-1.0.1.jar
[ERROR] urls[34] = file:/Users/PDurbin/.m2/repository/com/google/guava/listenablefuture/9999.0-empty-to-avoid-conflict-with-guava/listenablefuture-9999.0-empty-to-avoid-conflict-with-guava.jar
[ERROR] urls[35] = file:/Users/PDurbin/.m2/repository/com/google/code/findbugs/jsr305/3.0.2/jsr305-3.0.2.jar
[ERROR] urls[36] = file:/Users/PDurbin/.m2/repository/org/checkerframework/checker-qual/3.33.0/checker-qual-3.33.0.jar
[ERROR] urls[37] = file:/Users/PDurbin/.m2/repository/com/google/errorprone/error_prone_annotations/2.18.0/error_prone_annotations-2.18.0.jar
[ERROR] urls[38] = file:/Users/PDurbin/.m2/repository/com/google/j2objc/j2objc-annotations/2.8/j2objc-annotations-2.8.jar
[ERROR] urls[39] = file:/Users/PDurbin/.m2/repository/net/jodah/failsafe/2.4.0/failsafe-2.4.0.jar
[ERROR] urls[40] = file:/Users/PDurbin/.m2/repository/org/apache/commons/commons-lang3/3.6/commons-lang3-3.6.jar
[ERROR] urls[41] = file:/Users/PDurbin/.m2/repository/org/apache/commons/commons-text/1.1/commons-text-1.1.jar
[ERROR] urls[42] = file:/Users/PDurbin/.m2/repository/org/apache/httpcomponents/httpclient/4.5.13/httpclient-4.5.13.jar
[ERROR] urls[43] = file:/Users/PDurbin/.m2/repository/org/apache/httpcomponents/httpcore/4.4.13/httpcore-4.4.13.jar
[ERROR] urls[44] = file:/Users/PDurbin/.m2/repository/commons-logging/commons-logging/1.2/commons-logging-1.2.jar
[ERROR] urls[45] = file:/Users/PDurbin/.m2/repository/commons-codec/commons-codec/1.11/commons-codec-1.11.jar
[ERROR] urls[46] = file:/Users/PDurbin/.m2/repository/org/apache/maven/plugins/maven-assembly-plugin/3.1.0/maven-assembly-plugin-3.1.0.jar
[ERROR] urls[47] = file:/Users/PDurbin/.m2/repository/org/sonatype/sisu/sisu-inject-bean/1.4.2/sisu-inject-bean-1.4.2.jar
[ERROR] urls[48] = file:/Users/PDurbin/.m2/repository/org/sonatype/sisu/sisu-guice/2.1.7/sisu-guice-2.1.7-noaop.jar
[ERROR] urls[49] = file:/Users/PDurbin/.m2/repository/org/sonatype/aether/aether-util/1.7/aether-util-1.7.jar
[ERROR] urls[50] = file:/Users/PDurbin/.m2/repository/org/codehaus/plexus/plexus-component-annotations/1.5.5/plexus-component-annotations-1.5.5.jar
[ERROR] urls[51] = file:/Users/PDurbin/.m2/repository/org/sonatype/plexus/plexus-sec-dispatcher/1.3/plexus-sec-dispatcher-1.3.jar
[ERROR] urls[52] = file:/Users/PDurbin/.m2/repository/org/sonatype/plexus/plexus-cipher/1.4/plexus-cipher-1.4.jar
[ERROR] urls[53] = file:/Users/PDurbin/.m2/repository/org/apache/maven/shared/maven-common-artifact-filters/3.0.1/maven-common-artifact-filters-3.0.1.jar
[ERROR] urls[54] = file:/Users/PDurbin/.m2/repository/org/apache/maven/shared/maven-shared-utils/3.1.0/maven-shared-utils-3.1.0.jar
[ERROR] urls[55] = file:/Users/PDurbin/.m2/repository/org/apache/maven/shared/maven-artifact-transfer/0.9.0/maven-artifact-transfer-0.9.0.jar
[ERROR] urls[56] = file:/Users/PDurbin/.m2/repository/org/codehaus/plexus/plexus-archiver/3.5/plexus-archiver-3.5.jar
[ERROR] urls[57] = file:/Users/PDurbin/.m2/repository/org/iq80/snappy/snappy/0.4/snappy-0.4.jar
[ERROR] urls[58] = file:/Users/PDurbin/.m2/repository/org/tukaani/xz/1.6/xz-1.6.jar
[ERROR] urls[59] = file:/Users/PDurbin/.m2/repository/org/apache/maven/shared/file-management/3.0.0/file-management-3.0.0.jar
[ERROR] urls[60] = file:/Users/PDurbin/.m2/repository/org/apache/maven/shared/maven-shared-io/3.0.0/maven-shared-io-3.0.0.jar
[ERROR] urls[61] = file:/Users/PDurbin/.m2/repository/commons-io/commons-io/2.5/commons-io-2.5.jar
[ERROR] urls[62] = file:/Users/PDurbin/.m2/repository/org/apache/maven/shared/maven-filtering/3.1.1/maven-filtering-3.1.1.jar
[ERROR] urls[63] = file:/Users/PDurbin/.m2/repository/org/sonatype/plexus/plexus-build-api/0.0.7/plexus-build-api-0.0.7.jar
[ERROR] urls[64] = file:/Users/PDurbin/.m2/repository/org/codehaus/plexus/plexus-io/3.0.0/plexus-io-3.0.0.jar
[ERROR] urls[65] = file:/Users/PDurbin/.m2/repository/org/apache/maven/maven-archiver/3.2.0/maven-archiver-3.2.0.jar
[ERROR] urls[66] = file:/Users/PDurbin/.m2/repository/org/codehaus/plexus/plexus-utils/3.1.0/plexus-utils-3.1.0.jar
[ERROR] urls[67] = file:/Users/PDurbin/.m2/repository/org/bouncycastle/bcpkix-jdk15on/1.65/bcpkix-jdk15on-1.65.jar
[ERROR] urls[68] = file:/Users/PDurbin/.m2/repository/org/bouncycastle/bcprov-jdk15on/1.65/bcprov-jdk15on-1.65.jar
[ERROR] urls[69] = file:/Users/PDurbin/.m2/repository/org/codehaus/plexus/plexus-interpolation/1.24/plexus-interpolation-1.24.jar
[ERROR] urls[70] = file:/Users/PDurbin/.m2/repository/org/yaml/snakeyaml/1.32/snakeyaml-1.32.jar
[ERROR] Number of foreign imports: 1
[ERROR] import: Entry[import  from realm ClassRealm[project>edu.harvard.iq:dataverse:6.4, parent: ClassRealm[maven.api, parent: null]]]
[ERROR] 
[ERROR] -----------------------------------------------------
[ERROR] 
[ERROR] -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/PluginExecutionException
```